model/registry/gen-ai.yaml

```diff
@@ -71,6 +71,26 @@ groups:
         brief: The number of tokens used in the LLM response (completion).
         examples: [180]
         tag: llm-generic-response
+      - id: usage.token_type
```

Comment:
not blocking: throwing in some alternative attribute names (in case anyone has preference)

`gen_ai.usage.direction`
`gen_ai.token.direction`
`gen_ai.token.kind`
`gen_ai.token.type`
`gen_ai.token.source` `prompt|completion` or `client|model` or `request|response`